sktime/benchmarking/run.ipynb

```diff
@@ -0,0 +1,101 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b99fe5e9",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "145d5a96",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from kotsu import run\n",
+    "\n",
+    "from sktime.benchmarking import estimator_registry, validation_registry"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2a84db36",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "validation_registry.all()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "023bef3d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "estimator_registry.all()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "637b1a67",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run.run(estimator_registry, validation_registry, \"./results.csv\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "85c1d549",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv(\"./results.csv\")"
```

Comment:
Ideally, we'd want the result to be returned in memory as a DF

Reply:
I just did a new release of `kostu` this morning, v0.3.1, the `run` func now returns the results df :)